py-rse/bash-basics.Rmd

```diff
@@ -302,7 +302,7 @@ $ ls -F zipfs-law
 ```
 
 ```text
-bin/  data/  Makefile  makefiles/
+data/
```

Comment:
Does it really only contain the `data` folder at this point? (Looking down further: yup, it appears so.)